conf/agent/agent.conf

```diff
@@ -8,7 +8,7 @@ agent {
     server_port = "8081"
     socket_path ="/tmp/agent.sock"
     trust_bundle_path = "./conf/agent/dummy_root_ca.crt"
-    trust_domain = "example.org"
+    trust_domain = "eXAMPLe.org"
```

Comment:
This feels a little messy - perhaps a test case for the cli code instead?

Reply:
whoops! didn't mean for this to make it in.